canvas/cli/assignment/fetch.py

```diff
@@ -5,12 +5,11 @@
 import canvas.cli.common
 import canvas.config
 
-DEFAULT_TABLE = False
 DEFAULT_SKIP_HEADERS = False
 DEFAULT_SKIP_DESCRIPTION = False
 
-def run_cli(assignment = None, table = DEFAULT_TABLE, skip_headers = DEFAULT_SKIP_HEADERS,
-        skip_description = DEFAULT_SKIP_DESCRIPTION, **kwargs):
+def run_cli(assignment = None, table = canvas.cli.common.DEFAULT_TABLE, skip_headers = DEFAULT_SKIP_HEADERS,
+        skip_description = DEFAULT_SKIP_DESCRIPTION, output_json = canvas.cli.common.DEFAULT_JSON, **kwargs):
```

Comment:
Since `table`, `skip_headers`, and `skip_description` are not explicitly used (and only passed to canvas.cli.common.cli_list(),
we could get away with not specifying them (letting them live in `kwargs`) and then passing `**kwargs` to canvas.cli.common.cli_list().